### DISCUSSION TITLE
bigscape network_files not created

### DISCUSSION THREAD

**Discussion by tmesut** [medema-group/BiG-SCAPE#21] [Q&A]
I use bigscape.py in my python codes that are in a pipeline. Sometimes, Bigscape works perfectly but sometimes it doesn't create any network_files although the other files are generated.. After this situation, when I try bigscape.py on my terminal manually with parameters in parametre.txt file that was created by bigscape, it works. Unfortunately I cannot find the reason for this situation. Do you have any suggestions or ideas?

**Comment by jorgecnavarrom** [comment]
Unfortunately, I can't think of anything... if using the command in parameters.txt didn't work, I would think on e.g. too strict a cutoff. But in this situation I don't really know... Can you somehow redirect BiG-SCAPE's output in the pipeline? perhaps we could see some message there that could help...
